src/tests/AllCurrenciesRender.test.js

```diff
@@ -0,0 +1,22 @@
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+// eslint-disable-next-line import/extensions
+import store from '../Redux/store';
+
+import AllCurrencies from '../Components/AllCurrencies';
```

Comment:
- [ ] Great job so far, I think while making the changes, some imports are not working and the test cases are failing please see :point_down: please fix these :+1: 
![image](https://github.com/sameedmohsin28/currency-exchange-rate/assets/97521557/0b5b6abf-e024-440e-86b9-9b7a0f625183)

Reply:
The tests are working perfectly fine without making any changes here at my side. Could you please check and confirm once again on your side?
![image](https://github.com/sameedmohsin28/currency-exchange-rate/assets/122236545/d00a6b5c-612a-4d38-8836-ff62932b69b9)

Reply:
@PinkMoon25 
The tests are working perfectly fine without making any changes here at my side. Could you please check and confirm once again on your side?
![image](https://github.com/sameedmohsin28/currency-exchange-rate/assets/122236545/51b1047d-0eb4-4626-973e-ac342f1e7764)

Reply:
hi, It still does not pass, and the simple reason is you are importing the store like `import store from '../Redux/store'` notice the capital 'R' while your redux folder starts with a small 'r' replacing the import with `import store from '../redux/store'` solves the issue